scripts/generate-integration-pages.ts

```diff
@@ -178,6 +179,47 @@ function absoluteLinks({ base }: { base: string }) {
 	};
 }
 
+/** Remark plugin to replace GitHub . */
+function replaceAsides() {
+	return function transform(tree: Root) {
+		// visit blockquote
+		visit(tree, 'blockquote', (node) => {
+			// first child (<blockquote><p>...</p></blockquote>)
+			const [firstChild, trailingText] = node.children[0].children;
```

Comment:
It can be a bit hard to follow all the nested `children` in remark plug-ins, so I personally find it can help declaring variables so we get some more explicit names, like:

```suggestion
			const openingParagraph = node.children[0];
			const [firstChild, trailingText] = openingParagraph.children;
```